### PR TITLE
No need to stick to end of method

### PR DESCRIPTION
**Pull request by Dmitry-Me**
Clarify how exactly to decide where a call to the method should be located

### CODE REVIEW

File: src/mscorlib/src/System/GC.cs [hunk 1]
```diff
@@ -270,9 +270,12 @@ internal static int CollectionCount (int generation, bool getSpecialGCCount)
...
         //
-        // Users should insert a call to this method near the end of a
-        // method where they must keep an object alive for the duration of that
-        // method, up until this method is called.  Here is an example:
+        // Users should insert a call to this method right after the last line
+        // of their code where their code still needs the object to be kept alive.
+        // The object which reference is passed into this method will not
+        // be eligible for collection until the call to this method happens.
+        // Once the call to this method has happened the object may immediately
+        // become elegible for collection. Here is an example:
```
Comment (jkotas) [inline src/mscorlib/src/System/GC.cs]: `elegible` -> `eligible`

Comment (Dmitry-Me) [inline src/mscorlib/src/System/GC.cs]: @jkotas Fixed, replacing the commit.

**Comment by Dmitry-Me** [comment]
@jkotas I fixed that typo.

**Comment by jkotas** [comment]
@dotnet-bot test this please

**Comment by Dmitry-Me** [comment]
@jkotas LOL what? What could @dotnet-bot find in a comment-only change?

**Comment by jkotas** [comment]
Verify that it is really harmless comment only change, e.g. there is no accidental stray `.` in it. 

All builds failed on this one because of CI issues that we are having in last couple of days ... it is much easier and safer for me to just trigger re-test that manually try to verify everything.

**Comment by jkotas** [comment]
@dotnet-bot test this please

**Comment by Dmitry-Me** [comment]
@jkotas What's the deal with accidental stray `.`? I cannot find anything about it.

**Pull request merged**